### PR TITLE
upgrade dependents

### PR DESCRIPTION
**Pull request by skeeey**
1. upgrade kube to 1.26
2. upgrade controller-runtime to v0.14.5
3. upgrade addon-framework to v0.6.1

### CODE REVIEW

File: test/utils/utils.go [hunk 1]
```diff
@@ -273,13 +273,15 @@ func GetMetrics(metricPatterns ...string) []string {
...
 	if err != nil {
+		By("0---------------------->" + err.Error())
 		if err.Error() == "exit status 1" {
 			return []string{} // exit 1 indicates that grep couldn't find a match.
 		}
 
 		return []string{err.Error()}
 	}
 
+	By("1---------------------->" + string(matchingMetricsRaw))
```
Comment (dhaiducek) [inline test/utils/utils.go:284]: I'm assuming this is a debugging line?
```suggestion
```

**Review by dhaiducek — CHANGES_REQUESTED** [review]
Thanks for the update! Please see my review comments.

File: test/utils/utils.go [hunk 2]
```diff
@@ -273,13 +273,15 @@ func GetMetrics(metricPatterns ...string) []string {
 
 	matchingMetricsRaw, err := cmd.Output()
 	if err != nil {
+		By("0---------------------->" + err.Error())
```
[the remark this answers is not in the document]
Comment (skeeey) [reply test/utils/utils.go:276]: yes,  the e2e is failed, I'm investigating what's happed

[the remark this answers is not in the document]
Comment (dhaiducek) [reply test/utils/utils.go:276]: Sounds good--let me know if you have questions or would like me to also take a look!

[the remark this answers is not in the document]
Comment (skeeey) [reply test/utils/utils.go]: removed

File: test/utils/utils.go (hunk 1 above)
Comment (skeeey) [reply test/utils/utils.go]: removed

**Review by dhaiducek — APPROVED** [review]
Thanks for the update, @skeeey!

**Pull request merged**